src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -62,7 +72,7 @@ export function RegisterForm() {
       validateOnChange={true}
       validateOnBlur={true}
       validate={validate}
-      initialValues={{ email: '', name: '', password: '', surname: '' }}
+      initialValues={{ email: '', name: '', password: '', roomNumber: 0, surname: '' }}
```

Comment:
`roomNumber: 0` I think with this approach you are going to get rid of input placeholder

Reply:
It's hard to get rid of placeholder values complately as Formik requires some type of interface in <> brackets.